x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/searchablesnapshots/SearchableSnapshotShardStats.java

```diff
@@ -169,8 +171,9 @@ public CacheIndexInputStats(String fileName, long fileLength, long openCount, lo
         }
 
         CacheIndexInputStats(final StreamInput in) throws IOException {
-            this.fileName = in.readString();
-            this.fileLength = in.readVLong();
+            this.fileExt = in.readString();
```

Comment:
Similarly here, we should throw an `UnsupportedOperationException` for 7.11 nodes.